frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -101,18 +105,36 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
     const imageStream = builderImages[values.image.selected].obj;
 
     const {
+      application: { name: applicationName },
       project: { name: projectName },
     } = values;
 
+    const kNativeServiceUtil = (dockerImageRepository: string) => {
```

Comment:
This belongs in create resources so that any branching of resource creation is done in one spot outside this component.

Reply:
moved to `createResources`